contracts/root/staking/StakeManagerStakingData.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+
+/**
+ * @title StakeManagerStakingData
+ * @notice Holds all staking related data.
+ * Note that this is contract is designed to be included in StakeManager. It is upgradeable.
+ */
+abstract contract StakeManagerStakingData {
+    uint256 internal theTotalStake;
+    // validator => child => amount
+    mapping(address => mapping(uint256 => uint256)) private stakes;
+    // child chain id => total stake
+    mapping(uint256 => uint256) private totalStakePerChild;
+    // validator address => stake across all child chains.
+    mapping(address => uint256) private totalStakes;
+    // validator address => withdrawable stake.
+    mapping(address => uint256) private withdrawableStakes;
+
+    // Storage gap 
+    // solhint-disable-next-line var-name-mixedcase
+    uint256[1000] private __StorageGapStakeManagerStakingData;
```

Comment:
Same as above.

Reply:
As per above: Agree to switch to 50, and move to the end of the file. However, I think keeping a unique name helps with debug.

Reply:
@gretzke wdyt?

Reply:
the entire repo uses `__gap` I would keep it for consistency, if we decide to rename it we should do it everywhere